### DISCUSSION TITLE
Passing parameters from Kicad's PCB Editor

### DISCUSSION THREAD

**Discussion by SerraCabo** [freerouting/freerouting#397] [Q&A]
Hi.

Integration of Freerouting into PCB Editor works for me but I can't figure out how to activate Auto-router settings: fanout and optimization from within PCB Editor.

It works if I do it within Freerouting but as soon as I close Freerouting it forgets what I have selected.

Thanks for help
SC

**Comment by pka4916** [comment]
same problem here..    driving me nuts, and seems like there is no support either
